### PR TITLE
Feature/#31 student feedback

### LINKED ISSUE #31: [ 3 ] As a student, I can submit feedback during a lecture.
**AC**:
- After submitting a feedback, it is persisted in the backend of the model.
- The feedback is linked to the lecture it was submitted for
- The author of the feedback is not persisted with the feedback, to maintain anonymity

_Hint:
Use the [current paper prototype](https://www.figma.com/file/78SUuNyEDnrKGGeJPmvnmp/Paper-Prototype-1?node-id=0%3A1) as guidance, but remember that there will be a new version with a dashboard by wednesday._

### PR DESCRIPTION
**Pull request by m0e33**
- After submitting a feedback, it is persisted in the backend of the model.
- The feedback is linked to the lecture it was submitted for
- The author of the feedback is not persisted with the feedback, to maintain anonymity

Closes #31

### CODE REVIEW

File: app/controllers/feedbacks_controller.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+class FeedbacksController < ApplicationController
```
Comment (Jasperhino) [inline app/controllers/feedbacks_controller.rb]: Warum plural? Ist das nicht eher ein FeedbackController

File: app/helpers/feedbacks_helper.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+module FeedbacksHelper
```
Comment (Jasperhino) [inline app/helpers/feedbacks_helper.rb]: Plural?

File: app/controllers/feedbacks_controller.rb (hunk 1 above)
Comment (m0e33) [reply app/controllers/feedbacks_controller.rb]: Das hat das generate so benannt, sollte also passen.

Comment (m0e33) [reply app/controllers/feedbacks_controller.rb]: Look at this https://guides.rubyonrails.org/getting_started.html#adding-a-second-model

File: app/helpers/feedbacks_helper.rb (hunk 2 above)
Comment (m0e33) [reply app/helpers/feedbacks_helper.rb]: See above

**Pull request merged**